packages/backend/src/graphql/schema.graphql

```diff
@@ -917,6 +922,27 @@ input CreateOutgoingPaymentInput {
   idempotencyKey: String
 }
 
+input CreateOutgoingPaymentFromIncomingPaymentInput {
+  "Id of the wallet address under which the outgoing payment will be created"
+  walletAddressId: String!
+  "Incoming payment url to create the outgoing payment from"
+  incomingPayment: String!
```

Comment:
I think incomingPayment is good here, but do you think we should also change the `receiver` in the `CreateQuoteInput`?

i.e. should we move away from `receiver` altogether?